assets/src/design-system/components/button/buttonAsLink.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2020 Google LLC
```

Comment:
```suggestion
 * Copyright 2021 Google LLC
```
Good choice for design system component